packages/nimble-components/src/rich-text/editor/styles.ts

```diff
@@ -219,6 +219,10 @@ export const styles = css`
         place-items: center;
     }
 
+    nimble-toggle-button::part(start) {
```

Comment:
Do we need a test case to check whether tab index gets updated?

Reply:
We already have a test case to check the tab index value with the click of every button (linked below). Given that clicking on the icon is a specific action and the test cases are written for the entire button, they passed earlier.

https://github.com/ni/nimble/blob/abc7972d954e3b5335aaac27084ccf4519c9910f/packages/nimble-components/src/rich-text/editor/tests/rich-text-editor.spec.ts#L203C1-L205C31

Reply:
As discussed, I queried the slot element of the icon and checked the tab index value of the button on click.